webapp/channels/src/components/admin_console/server_logs/logs.tsx

```diff
@@ -181,6 +209,7 @@ export default class Logs extends React.PureComponent<Props, State> {
                         />
                     </button>
                 </div>
+                {ToggleView}
```

Comment:
If JSON logs are disabled, this toggle should not be shown as the Formatted page will be empty.

Reply:
Thanks for your comments @hanzei . I have updated the code base. Can you please review it again?